eos/maths/polylog.cc

```diff
@@ -15,6 +33,7 @@ namespace eos
 
     namespace dilog_impl
     {
+        // for the dilogarithm the coefficients of f1 are : Zeta(2 - m) / m! starting from m = 0 to m = max_iterations
```

Comment:
Why is this comment here and not above `f1`?

Reply:
the list in the line below contains the series coefficients of f1

Reply:
I agree with @vikusc here, the comment is well placed.